docs/source/host-collect-analyze/blockDevices.md

```diff
@@ -0,0 +1,108 @@
+---
+title: Block Devices
+description: Collect and analyze information about the host system block devices.
+---
+
+## Block Devices Collector
+
+The `blockDevices` collector can collect information about all of the block devices on a host.
+
+### Parameters
+
+`None`
```

Comment:
Looks like they don;t literally type `None`, yes? If so, I suggest changing this to either:

None.

or

None (`{}`).